uiowa/uiowa
# Patch core to allow saved changes to embedded media

Potentially resolves #1335 

## To test

- Make sure stage file proxy is uninstalled locally.
- Add media, save. Go back in and try changing view modes. Save. Update layout block.

The view mode should update and when you go back into the wysiwyg edit media modal, the view mode you selected should still be selected.
